source/javascripts/app/_search.js

```diff
@@ -1,11 +1,13 @@
 //= require ../lib/_lunr
 //= require ../lib/_jquery
 //= require ../lib/_jquery.highlight
-(function () {
+;(function () {
```

Comment:
Is the ";" in front of the function a typo or it's meant to be there?

Reply:
No, it's an error,  it mustn't be here.